lib/shopify/customer.ex

```diff
@@ -39,6 +39,32 @@ defmodule Shopify.Customer do
     }
   end
 
+  @doc """
+  Create a metafield for a customer.
+
+  ## Examples
+
+    iex> session = Shopify.new_private_session("mystore", "xxx...xxx", "yyy...yyy")
```

Comment:
No need to add session init in examples. It's beyond the scope of what the example should cover.